integration-tests/bats/migration-integration.bats

```diff
@@ -29,7 +29,7 @@ teardown() {
     dolt tag -v
     run dolt tag -v
     [ "$status" -eq 0 ]
-    [[ "$output" =~ "tdkt7s7805k1ml4hu37pm688g5i0b8ie" ]] || false
+    [[ "$output" =~ "7jrvg1ajcdq6t9sevcejv4e9o0fgrmle" ]] || false
```

Comment:
So for this test (and the following one), I'm assuming you ran these commands locally and got the new hashes, so it's not `tdkt7s7805k1ml4hu37pm688g5i0b8ie`, but `7jrvg1ajcdq6t9sevcejv4e9o0fgrmle` now. That's fine. But you're not replacing all instances of `tdkt7s7805k1ml4hu37pm688g5i0b8ie`. It's probably easier to reverse this file, and just do a find->replace on that exact hash. That will fix your test failure.